crates/bitwarden-exporters/src/cxf/ssh.rs

```diff
@@ -0,0 +1,99 @@
+use bitwarden_ssh::{error::SshKeyImportError, import::import_der_key};
+use bitwarden_vault::FieldType;
+use credential_exchange_format::SshKeyCredential;
+
+use crate::{cxf::editable_field::create_field, Field, SshKey};
+
+/// Convert SSH key credentials to SshKey and custom fields
+pub(super) fn to_ssh(
+    credential: &SshKeyCredential,
+) -> Result<(SshKey, Vec<Field>), SshKeyImportError> {
+    // Convert to OpenSSH format
+    let encoded_key: Vec<u8> = credential.private_key.as_ref().into();
+    let encoded_key = import_der_key(&encoded_key)?;
+
+    let ssh = SshKey {
+        private_key: encoded_key.private_key,
+        public_key: encoded_key.public_key,
+        fingerprint: encoded_key.fingerprint,
+    };
+
+    let fields = [
+        credential.key_comment.as_ref().map(|comment| Field {
```

Comment:
Note (non-actionable): In the agent, we treat the name of the cipher as the comment, and advertise it as such. We may want to research (not here) how other clients treat comment vs name, to not disrupt user experience. If someone comes over from another product which uses a separate key comment field, suddenly all their ciphers may be named something different.

Reply:
I think we will have a lot of follow up work as other credential managers implements credential exchange and we discover edge cases 😰